src/Microsoft.Azure.WebJobs.Extensions.Kafka/Diagnostics/KafkaEventInstrumentation.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.Diagnostics
+{
+    internal static class KafkaEventInstrumentation
+    {
+        // For Trigger
+        // Try to extract traceparent header
+        public static bool TryExtractTraceParentId(IKafkaEventData kafkaEvent, out string traceParentId)
```

Comment:
bool is not being used anywhere so I think it's return type should be void

Reply:
Done. Changed the type to void.